man/tpm2_nvdefine.1.md

```diff
@@ -36,7 +36,7 @@
     entitiy. Either the raw bitfield mask or "nice-names" may be used. See
     section "NV Attributes" for more details.
 
-  * **-P**, **--auth-hierarchy**=_HANDLE\_PASSWORD_:
+  * **-P**, **--hierarchy**=_HANDLE\_PASSWORD_:
```

Comment:
the tool is wrong here, need to update tpm2_nvdefine.c

Reply:
@williamcroberts right, I made that comment for tpm2_nvread but I actually meant for tpm2_nvdefine.